sdk/datafactory/Microsoft.Azure.Management.DataFactory/src/Generated/Models/GoogleAdWordsLinkedService.cs

```diff
@@ -76,9 +79,10 @@ public GoogleAdWordsLinkedService()
         /// authentication. Credentials are encrypted using the integration
         /// runtime credential manager. Type: string (or Expression with
         /// resultType string).</param>
-        public GoogleAdWordsLinkedService(object clientCustomerID, SecretBase developerToken, string authenticationType, IDictionary<string, object> additionalProperties = default(IDictionary<string, object>), IntegrationRuntimeReference connectVia = default(IntegrationRuntimeReference), string description = default(string), IDictionary<string, ParameterSpecification> parameters = default(IDictionary<string, ParameterSpecification>), IList<object> annotations = default(IList<object>), SecretBase refreshToken = default(SecretBase), object clientId = default(object), SecretBase clientSecret = default(SecretBase), object email = default(object), object keyFilePath = default(object), object trustedCertPath = default(object), object useSystemTrustStore = default(object), object encryptedCredential = default(object))
```

Comment:
This is a breaking change. Either provide a customization (see https://github.com/Azure/azure-sdk-for-net/tree/main/sdk/compute/Microsoft.Azure.Management.Compute/src/Customizations as a reference), or you need to bump the major version to `6.0.0.0`.